### ISSUE TITLE
Name overlap: coinValue

### ISSUE THREAD

**Issue by m0dev76** [haskoin/haskoin#67]
```
Ambiguous occurrence `coinValue'
    It could refer to either `Network.Haskoin.Transaction.coinValue',
                             imported from `Network.Haskoin.Transaction' at script/hw.hs:109:1-34
                             (and originally defined in `haskoin-0.0.2.1:Network.Haskoin.Transaction.Builder')
                          or `Network.Haskoin.Stratum.coinValue',
                             imported from `Network.Haskoin.Stratum' at script/hw.hs:110:1-30
                             (and originally defined in `haskoin-0.0.2.1:Network.Haskoin.Stratum.Types')
```

**Comment by jprupp** [comment]
I'll change the Stratum code accordingly.

**Comment by m0dev76** [comment]
Fixed in [#72]
